examples/getting_started/olive-awq-ft-llama.ipynb

```diff
@@ -0,0 +1,291 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "tv6vx7wooDfk"
+   },
+   "source": [
+    "# ✨ Quantize & Finetune an SLM with Olive\n",
+    "\n",
+    "> ⚠️ **This notebook will quantize an Small Language Model (SLM) using the AWQ algorithm, which requires an Nvidia A10 or A100 GPU device.**\n",
+    "\n",
+    "In this notebook, you will:\n",
+    "\n",
+    "1. Quantize Llama-3.2-1B-Instruct model using the [AWQ Algorithm](https://ar5iv.labs.arxiv.org/html/2306.00978).\n",
+    "1. Fine-tune the quantized model to classify English phrases into Surprise/Joy/Fear/Sadness.\n",
+    "1. Optimize the fine-tuned model for the ONNX Runtime.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 🐍 Install Python dependencies\n",
+    "\n",
+    "The following cells create a pip requirements file and then install the libraries."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%writefile requirements.txt\n",
+    "\n",
+    "olive-ai==0.7.1\n",
+    "transformers==4.44.2\n",
+    "autoawq==0.2.6\n",
+    "optimum==1.23.1\n",
+    "peft==0.13.2\n",
+    "bitsandbytes==0.44.1\n",
```

Comment:
nit: bitsandbytes is not needed for Lora.

Reply:
Good spot... Removed.